examples/express.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
```

Comment:
Line space please

Reply:
should't this be reflected in eslint rules?